src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(100);

```

Comment:
1) Random object creates every time you call getRandomColor. Think of the approach, where you create an object of Random only once for this class
2) ColorSupplier object creates every time you call getRandomBall. Think of approach, when you create an object of ColorSupplier only once for this class
3) random.nextInt() returns 0 inclusively and 100 exclusively